src/wallet/context.h

```diff
@@ -10,6 +10,8 @@ namespace interfaces {
 class Chain;
 } // namespace interfaces
 
+struct NodeContext;
+
```

Comment:
Unnecessary line. Could be removed.

Reply:
Without it I get:

```
In file included from wallet/context.cpp:5:
./wallet/context.h:28:5: error: ‘NodeContext’ does not name a type; did you mean ‘WalletContext’?
   28 |     NodeContext* nodeContext{nullptr};
      |     ^~~~~~~~~~~
      |     WalletContext
```

However I have switched this out for the full include of `node/context.h` and that should be cleaner, thanks for the feedback.

Reply:
Yep, no worries. Looks good for now.